package lab9.backend.application.port.`in`.signup

class UserAlreadyExistsException: Exception("User with such name already exists")